codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/awssdk/shapevisitor/AwsSdkToDafnyShapeVisitor.java

```diff
@@ -434,28 +434,21 @@ var enum interface{}
       }
 
       if (shape.hasTrait(DafnyUtf8BytesTrait.class)) {
-        writer.addUseImports(SmithyGoDependency.stdlib("unicode/utf8"));
+        throw new UnsupportedOperationException(
+          "Dafny utf8bytes trait is not supported in aws sdk models: " +
+          shape.toShapeId().getName()
+        );
       }
-      final var underlyingType = shape.hasTrait(DafnyUtf8BytesTrait.class)
-        ? """
-            dafny.SeqOf(func () []interface{} {
-            utf8.ValidString(%s%s)
```

Comment:
Just wondering, before encoding to utf16, don't we have to check if its a valid utf character?

Reply:
The value will always be a valid string (replacement char is a valid string). In any case, I merged in the changes to use the Std Lib extern to be consistent.